Dockerfile

```diff
@@ -5,4 +5,4 @@ WORKDIR /app
 RUN pip3 install -r requirements.txt
 RUN find . -name "*.pyc" | xargs rm -rf
```

Comment:
it's probably better practice to put this cleanup in a .dockerignore (one less layer)